Firestore/test.sh

```diff
@@ -41,18 +41,14 @@ test_iOS() {
 test_CMake() {
   echo "cpu core: $(sysctl -n hw.ncpu)"
   echo "set cmake build" && \
-    mkdir build && \
+    mkdir -p build && \
     cd build && \
     cmake .. || \
     exit 1
 
   echo "initial cmake build" && \
```

Comment:
might update the message as well (i.e. not just initialize but test as well)

Reply:
Good idea, done.